lib/controllers/ServiceFabrikApiController.js

```diff
@@ -296,7 +295,7 @@ class ServiceFabrikApiController extends BaseController {
         if (metadata.state !== 'succeeded') {
           throw new UnprocessableEntity(`Can not restore backup '${backup_guid}' due to state '${metadata.state}'`);
         }
-        if (metadata.plan_id !== plan_id) {
+        if (!req.manager.isRestorePossible(metadata.plan_id)) {
           throw new UnprocessableEntity(`Backup '${backup_guid}' to restore from is not applicable to the instance's service plan`);
```

Comment:
sentence in the error does not sound right gramatically. Can you please check.

Reply:
Updated